main.py

```diff
@@ -24,54 +25,7 @@
 # For example, if set to 120, the bot will introduce a random delay between 1 and 120 seconds each time it rechecks.
 MaxRandomDelay = 120
 
-# Accounts will be checked in the order they are listed
-AccountList = [
```

Comment:
The reason I didn't move the `AccountLists` to another file is related to a YouTube video. This change might confuse new users, even though it is necessary. I'm not sure whether to proceed with it or not!